src/app/pages/send/send-crypto-asset-form/hooks/use-send-form-navigate.ts

```diff
@@ -82,8 +84,7 @@ export function useSendFormNavigate() {
         name,
         tx,
       }: ConfirmationRouteStacksSip10Args) {
-        return navigate('confirm', {
-          replace: true,
+        return navigate(`${location.pathname}/confirm`, {
```

Comment:
The previously working relative paths now fail so I am using the `location.pathname` to prepend this route. 

I think this may have stopped working as to move the `Continue` button to the footer, we had to change the form to use `props.handleSubmit()`

e.g. in `BtcSendForm`:

```
  return (
            <Form>
              <Card
                footer={
                  <Footer variant="card">
                    <Button
                      data-testid={SendCryptoAssetSelectors.PreviewSendTxBtn}
                      onClick={() => props.handleSubmit()}
                      type="submit"
                    >
                      Continue
                    </Button>
                    <AvailableBalance balance={formatMoney(btcBalance.balance)} />
                  </Footer>
                }
              >
                <CardContent dataTestId={SendCryptoAssetSelectors.SendForm}>
                  <AmountField
    
```

Before we had a lot of nested components doing this that drilled down a lot to a `preview-button` component